configuration.yaml

```diff
@@ -8,13 +8,14 @@ experiment_prefix: standard
 ################################################################################
 
 # dataset
-################################################################################
+###########################################ß#####################################
 #chromosomes: ['chr1', 'chr2', 'chrX']
 chromosomes: ['chr10']
 segment_length: 2000
 overlap: 500
 num_classes: 5 
-num_queries: 10      
+num_queries: 10
+num_nucleobase_letters: 6      
```

Comment:
Maybe I'm forgetting this one but what is the 6th nucleobase letter used in our dataset, other than ["A", "C", "G", "T", "N"]?

Reply:
Those codes were borrowed from your previous [project](https://github.com/Ensembl/gene_pcp), I think the reason why the `num_nucleobase_letters` is 6 is that there is a padding letter.
https://github.com/yangtcai/Ensembl-Repeat-Identification/blob/0410964aa2ee88759d5f0776c2bcd260954f9eaf/dataloader.py#L251-L258